### ISSUE TITLE
Likeliness to get banned for third-party software

### ISSUE THREAD

**Issue by EmilyJen** [giorgi-o/SkinPeek#115]
Hi!

First of all I want to mention that this is an awesome tool that I've been using for some time now! So thanks for creating and maintaining it.

Yet, I wonder if anyone ever had their account banned for third-party software because of this tool. I know it's hard to pin down what exactly caused a ban due to Riots policy for not disclosing any information, but maybe someone had nothing else running besides this tool, that would atleast make it a more likely candidate.

Did someone maybe contact support about this matter? Or someone on the Riot team?

Another project, Deceive, has claimed that it's sort of (?) officially allowed, with a screenshot by, I presume, a dev saying that no one at Riot will go after someone for using Deceive. 

So the question is, how likely is one to get banned for using SkinPeek?

**Comment by giorgi-o** [comment]
That's an excellent question! This is quite a long post, so TLDR at the bottom.

 

The short answer is no, you won't get banned for using SkinPeek or any other store checker. I can say that with very high certainty. I have no way of knowing how many total SkinPeek users there are, but I would say ~100k is a conservative estimate - at its peak, Ministral (public SkinPeek instance) had 71k.  And yet, no one has been banned for using SkinPeek so far. I and many, many others have been using it daily for years with no problems whatsoever.

 

-------

 

To be clear, Riot could easily implement a SkinPeek detection algorithm if they wanted to (their aimbot detection algorithm is already 100x more complex). But it makes no sense for them to do so. Yes, SkinPeek is _technically_ against ToS, which bans "any third party application". But in my mind, that is only written in there for a few reasons:

 

- So that they don't get sued for banning people that use actually harmful programs (e.g. instalocking bots)

- To discourage people from going around and installing random Valorant programs all over the internet

- So that if a program dev messes up and, say, leaks everyone's passwords online, Riot can say "it's against ToS guys, not our responsibility!"

 

But as for actively seeking out anyone using a store checker and banning them, that doesn't make sense for them to do. They know full well these are programs written by hobby devs in their spare time for fun (no one's making money off SkinPeek). Both the users and the devs are just passionate Valorant players, it would be stupid to suddenly ban them.

 

Plus in the case of SkinPeek specifically, think about it: the people checking their store on their phone are often skin enthusiasts, who typically spend the most money on skins. Banning _them_ would be outright stupid.

 

If one day Riot decided they wanted people to stop using SkinPeek and other programs... well first they would just change the API and cause SkinPeek to stop working - no one uses it anymore, no ban needed. Or, back when they wanted people to stop using vRY and WAIUA, what they did was post a [tweet](https://twitter.com/PlayVALORANT/status/1539728676815642624) telling people to stop using it.

 

And in the case of vRY and WAIUA, they were programs that let you bypass streamer mode (among other things), i.e. actively degrading other people's experience. What's the worst you can do with a shop checker bot? Buy more skins than your friends?

 

--------

 

So Riot aren't going to ban anyone for using SkinPeek. However for full transparency, I should mention that about once a month, someone comes into the support server claiming they got perma-banned because of SkinPeek. All these people are adamant they never ever cheated, and didn't use any other programs, so it _had_ to be SkinPeek.

 

Since this happened so infrequently, I always assumed it was just that their account got hacked, and the hacker used an aimbot on their account. I just told them to contact support and didn't think much more of it. That was until [Mistral](https://x.com/mistralwz), the awesome guy hosting Ministral and basically my partner on SkinPeek development, was the next person to get the banhammer.

 

I know for a fact he would never even think use a game hack, so what does that mean... is SkinPeek to blame? Was it hosting it or using it that got him banned? Turns out, it was neither! After being stonewalled by Riot Support 3 times, he managed to get in touch with a Riot dev who told him it was a false positive... -_-

 

But relevant to this discussion was that he gave that Riot dev a list of everything he had installed on his PC that might have caused the ban, including AtlasOS, Deceive, and SkinPeek. The dev assured him that none of these programs caused the ban, and he could keep using them 👍

![image](https://github.com/giorgi-o/SkinPeek/assets/20621396/6c666d37-94bb-4f10-b79f-c263e1963beb)
 

More info about this incident [here](https://gist.github.com/mistralwz/142b901b333dc08e08aa524e0e077e2d) if you're curious.

 

_(Note: this just means it's confirmed that Vanguard doesn't detect SkinPeek as a game hack, not that it is officially allowed by Riot.)_

 

--------

 

**TLDR:** No one has been banned or even warned, and it makes no sense for Riot to stop you from buying more skins. A Riot dev even confirmed SkinPeek won't get you banned. So yes, it's safe to use.

**Comment by warriorzz** [comment]
Honestly, this issue deserves a pin, well written explanation and answers most frequently questions - something I would search for in the existing issues

**Comment by EmilyJen** [comment]
Hey, thanks a lot for the very detailed response! I agree with @warriorzz, this should definitely be pinned in some way :) Thanks a lot for all the work on this tool again ❤️

**Comment by giorgi-o** [comment]
I'll do one better, I've put it in a [wiki article](https://github.com/giorgi-o/SkinPeek/wiki/Can-I-get-banned-for-using-SkinPeek%3F) which I'll link towards in the Readme :)
